src/components/ModelDialog/components/ModalContent/styled.ts

```diff
@@ -0,0 +1,11 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2023 Gemeente Amsterdam
+import { themeSpacing } from '@amsterdam/asc-ui'
+import styled from 'styled-components'
+
+export const StyledSection = styled.div`
+  display: flex;
+  flex-direction: column;
+  height: 100%;
```

Comment:
Don't think think these three do anything right?

You also need to think about that people can pass anything in children, and they want to have their own styling, maybe that's not flexbox or columns.

Reply:
Actually, if it's not there the emailpreview doesn't show properly. I think that in the future if people want to make use of the modaldialog they'll need to use a flexbox and collumns. If it doesn't fit, then they can make use of the Modal. 
I've changed the heigth to flex-grow: 1.